tests/ui/unseparated_prefix_literals.rs

```diff
@@ -17,4 +21,8 @@ fn main() {
     let _okf2 = 1_f32;
     let _failf1 = 1.5f32;
     let _failf2 = 1f32;
+
+    // Counter example
+    let _ = line!();
```

Comment:
Ok it is indeed the `line!` macro. I think this is a special case, since it's an compiler built-in macro and nothing you could write in actual code. Just add a check for `utils::is_direct_expn_of("line")` (not 100% if the function name is correct, but a similar function exists)

Reply:
I checked locally, the `is_direct_expn_of` or `is_expn_of` doesn't help to detect `line!()` macro.

Reply:
Ok, I need to take a closer look on this then. I probably won't get to it before this weekend though..